backend/src/main/java/com/hackner/musicompass/discogsapi/model/DiscogsArtistSearchResults.java

```diff
@@ -15,6 +16,5 @@
 public class DiscogsArtistSearchResults {
 
     @JsonProperty("results")
```

Comment:
you don't need the json property here

Reply:
I see. I will keep it like this, if sth will be changed there...